include/mbedtls/config.h

```diff
@@ -232,27 +232,27 @@
 /**
  * \def MBEDTLS_DEPRECATED_WARNING
  *
- * Mark deprecated functions so that they generate a warning if used.
- * Functions deprecated in one version will usually be removed in the next
- * version. You can enable this to help you prepare the transition to a new
- * major version by making sure your code is not using these functions.
+ * Mark deprecated functions and features so that they generate a warning if
+ * used. Functionality deprecated in one version will usually be removed in the
+ * next version. You can enable this to help you prepare the transition to a
```

Comment:
Suggestion: "in the next _major_ version"?

Reply:
> "in one version will usually be removed in the next version."

That's not really how we work. Deprecated features seem to hang around a long time. This is pre-existing, but I think it would be better to say "a future version".

Entirely your choice @mpg.

Reply:
I agree but I'd rather not tweak the wording now, as we might soon have better-defined policies (hopefully).